rhombus/private/string.rkt

```diff
@@ -184,6 +187,11 @@
   (check-readable-string who s)
   (string->number s))
 
+(define/method (String.contains s1 s2)
+  #:inline
+  #:primitive (string-contains?)
+  (string-contains? s1 s2))
```

Comment:
`string-contains?` isn’t a “primitive”, because it’s defined in Racket.  This function will have to include two explicit checks.  (The idea of `#:inline` and `#:primitive` is that primitive functions defined in the kernel can be directly used.  The latter actually sets up the error rewriting, which only applies to actual primitive functions.)

Reply:
I was following what's going on with `String.length`, since that seemed to be close in form. What should this definition look like? This?

```
(define/method (String.contains s1 s2)
  #:inline
  (string-contains? s1 s2))
```

Or should I drop the `#:inline` too?

Reply:
It should look more like `String.to_number`, which also doesn’t use a (kernel) primitive.  There isn’t much benefit to inline wrappers of non-primitives (trivial ones *will* get inlined anyway, so we just trust the compiler for those).  You should also include explicit checks, because we want to avoid error reporting in the underlying implementation.